agent/utils/ttime/test_time.go

```diff
@@ -36,6 +37,7 @@ func NewTestTime() *TestTime {
 // Warp moves the mock time forwards by the given duration.
 func (t *TestTime) Warp(d time.Duration) {
 	t.warped += d
+	log.Criticalf("WARPED TIME: %v", d)
```

Comment:
The intent is for it to stand out in the logs, since `TestTime` should only ever be used during tests.